### ISSUE TITLE
[New App Shortcut]: Shaw BlueCurve TV

### ISSUE THREAD

**Issue by jsd58** [PRProd/HA-Firemote#132]
### App Name

Shaw BlueCurve TV

### Device Platform

Amazon Fire

### Application Info

TV app available in Canada for Shaw TV subscribers. In the past this app was also known as Shaw FreerangeTV

**Comment by PRProd** [comment]
Hello @jsd58!  Thanks for joining us and submitting your request!

Because I'm not in Canada, I was unable to download this app through the store, and I had to go about it a different way. I'd like your help to confirm that I have added it correctly.  If you are able to download the new pre-release version [v2.3.3-b2](https://github.com/PRProd/HA-Firemote/releases/tag/v2.3.3-b2) of Firemote and see if the Shaw BlueCurve TV button works as expected, that would be helpful.

Thanks!
